rust/main/config/mainnet_config.json

```diff
@@ -4029,7 +4029,7 @@
       "validatorAnnounce": "0xf57d954bf3ddb5f1032a0e020a99e931215cf83ceb4de987c781488065aaae0d",
       "gasPrice": {
         "denom": "ustrd",
-        "amount": "0.005"
+        "amount": "0.025"
```

Comment:
where is this coming from? ditto the osmosis change

Reply:
i still see 0.005 for stride on the cosmos registry here https://github.com/cosmos/chain-registry/blob/master/stride/chain.json

and 0.1 for osmosis here https://github.com/cosmos/chain-registry/blob/master/osmosis/chain.json

Reply:
just ran `yarn agent-configs`  locally as the ci was complaining, and that is the result i got, i'll double check what happened there